### ISSUE TITLE
Google Chrome Ending Support for Roam Toolkit

### ISSUE THREAD

**Issue by RussAlan** [roam-unofficial/roam-toolkit#225]
I have been using your wonderful Chrome extension Roam Toolkit for spaced repetition.  It has been the very foundation of my daily engagement with Roam for nearly four years.  Google is alerting me that, "This extension may soon no longer be supported because it doesn't follow best practices for Chrome extensions."   Do you have any plans to address this?  I havent found any other spaced repetition solution that fits my needs and I would prefer to continue using Roam ToolKit.  Thank you!

**Comment by Stvad** [comment]
They probably refer to the fact that it's still using Manifest v2. 

Specifically for the same approach to spaced repetition - you can use "Date UX improvements" plugin from the Roam Depot that I built, or a more elaborate version of that that you can load as roam-js plugin: https://x.com/VladyslavSitalo/status/1689888582771912704

**Comment by RussAlan** [comment]
Thank you Stvad.  I installed Date UX Improvements and it works perfectly.  Does this mean I no longer need the Chrome RoamToolKit plug-in and I can uninstall it?

**Comment by Stvad** [comment]
Yeah, though note that I think the version of the plugin in the Roam Depot doesn't have keyboard shortcuts, I think

**Comment by RussAlan** [comment]
You are correct.  No keyboard shortcuts in Roam Depot version.  OUCH.  Is there any way I can encourage you to add back the old shortcuts before Google kills Roam ToolKit?  Thanks again for reply.

**Comment by Stvad** [comment]
I recommend trying the experimental extension version, it has a bunch of other improvements (and also does keyboard shortcuts). I might get around to publishing it to depot at some point, but no specific plans yet

**Comment by vzaidman** [comment]
Ouch it was just shut down on me! Could I help somehow to restore it?

**Comment by RussAlan** [comment]
I echo vzaidman's cri de coeur of pain. Today Chrome permanently killed Roam ToolKit - my favorite Chrome extension.  Ugh.   The Roam Depot version has no shortcut keys and the pop-up window doesn't clear after you select an option making it slow and cumbersome to use. Would love to have the shortcut keys back. I press those shortcut keys at least a hundred times a day. I would pay for this plug-in!  Stvad, help please.

**Comment by RussAlan** [comment]
FYI: above I am referring to shortcut keys for AGAIN/HARD/GOOD/EASY in spaced repetition.   IMPORTANT QUESTION: I am now using "Date UX Improvements" plug-in from Roam Depot for spaced repetition, but how to you CREATE a new spaced repetition without the shortcut Shift+Cntrl+1?  There is nothing in the documentation about creation.   Thanks!

**Comment by Stvad** [comment]
The roam-js version of the extension I mention here https://x.com/VladyslavSitalo/status/1689888582771912704 has the keyboard shortcuts.

For creating - you can just give it any date (with `;1 week;` or `/Tomorrow` commands) and then you can reschedule it as any other srs item.

I'd be also open to a having a bounty to upgrade the roam depot version of extension to have shortcuts/or create a new extension in roam depot that is an evolution of this.

**Comment by Stvad** [comment]
@vzaidman what part of it's functionality are you using?

**Comment by RussAlan** [comment]
Thanks Stvad for the tip on adding a date. Perfect.  For whatever reason, I can't get the roam-js version to run in my graph with shortcut keys, in fact I can't get it to run at all, so I have retreated to the Roam Depot install of "Date UX improvements" for spaced repetition.  This works just fine but is missing the very handy shortcut keys for  AGAIN/HARD/GOOD/EASY.  Let me know what sort of bounty you have in mind.

**Comment by vzaidman** [comment]
> [@vzaidman](https://github.com/vzaidman) what part of it's functionality are you using?

The shortcut keys for spaced repetition and inline date manipulation.

**Comment by Stvad** [comment]
Not sure if anyone cares by this point, but I've updated the Date UX improvement extension to have shortcuts :) 
It's still pending review for roam/depot - you can track that here: https://github.com/Roam-Research/roam-depot/pull/1279

For people who were talking about bounty: you can do donations on GitHub Sponsors profile: https://github.com/sponsors/Stvad?frequency=one-time
